app/views/content/dates_and_deadlines.md

```diff
@@ -1,23 +1,13 @@
 ### <%= RecruitmentCycle.cycle_name(RecruitmentCycle.current_year) %> recruitment cycle – current
 | **Date and time** | **What happens** |
 | --- | --- |
-| <%= CycleTimetable.apply_opens.to_fs(:govuk_date_and_time) %> | Start of <%= RecruitmentCycle.cycle_name(RecruitmentCycle.current_year) %> recruitment cycle - candidates can apply for courses. |
-| <%= holidays[:christmas][:begins].to_fs(:govuk_date) %> to <%= holidays[:christmas][:ends].to_fs(:govuk_date) %> | This period is not counted as working days when calculating time to make a decision. |
-| <%= holidays[:easter][:begins].to_fs(:day_and_month) %> to <%= holidays[:easter][:ends].to_fs(:govuk_date) %> | This period is not counted as working days when calculating time to make a decision. |
-| <%= CycleTimetable.show_summer_recruitment_banner.to_fs(:govuk_date) %> | Time to make a decision is reduced from 40 working days to 20 working days. |
-| <%= CycleTimetable.apply_1_deadline.to_fs(:govuk_date_and_time) %> | Candidates can no longer apply for courses, unless they have already applied within this recruitment cycle. |
-| <%= CycleTimetable.apply_2_deadline.to_fs(:govuk_date_and_time) %> | Candidates can no longer apply for courses. |
-| <%= CycleTimetable.reject_by_default.to_fs(:govuk_date_and_time) %> | End of <%= RecruitmentCycle.cycle_name(RecruitmentCycle.current_year) %> recruitment cycle - applications awaiting decisions are automatically rejected. |
+| <%= CycleTimetable.apply_opens.to_fs(:govuk_date_and_time) %> | Start of <%= RecruitmentCycle.cycle_name(RecruitmentCycle.current_year) %> recruitment cycle. Candidates can apply for courses. |
```

Comment:
We're missing the date for Find opening above Apply opening.

Reply:
Pushed